static/js/index.js

```diff
@@ -5159,7 +5159,7 @@ function getTimeSince (date) {
     return str;
 }
 
-const ivFilterPrompt = 'Please enter an IV Filter. Example: (S0-1 & A15 & D15 & (CP1400-1500 | CP2400-2500)) | L35 | 90-100';
+const ivFilterPrompt = 'Please enter an IV and/or Level Filter. Examples:\n(A0-1 & D15 & S15 & (CP1400-1500 | CP2400-2500)) | L34-35 | 90-100\n!0-100\nA0 & D0 & S0 | 100';
```

Comment:
```suggestion
const ivFilterPrompt = 'Please enter an IV and/or Level Filter. Examples:\n(A0-1 & D15 & S15 & (CP1400-1500 | CP2400-2500)) | L34-35 | 90-100\nShows all Pokemon that are between CP1400-1500 & CP2400-2500, level 34 or 35, have 0 or 1 Attack, 15 Defense/Stamina, and total IVs are between 90 and 100%. \n!0-100\nDisplays Pokemon that do not have their IVs checked\nA0 & D0 & S0 | 100\nResults in all 0% or 100% Pokemon';
```

Reply:
The issue with this is that it adds more lines to the alert box, which might not be a problem, but smaller devices _might_ have an issue. I'll test the changes right now.

Reply:
Let me know. I think showing what all of that actually results in can be helpful for the users to process it, but fine to save it for a ui redo at some point if it's messy.

Reply:
Just a bit overwhelming lol
![image](https://user-images.githubusercontent.com/1327440/93657054-0336bc80-f9e4-11ea-8ea2-e04c5f6a04f3.png)

Reply:
this recommendation is too much for this UI element. 

💡 how about a "Help" button next to configure that links to [https://wwm.readthedocs.io/projects/mapjs/en/latest/filtering](). From there we can include 3-4 examples + english text on what it is actually doing?